engine/rpcserver.go

```diff
@@ -898,6 +904,8 @@ func (s *RPCServer) CancelOrder(ctx context.Context, r *gctrpc.CancelOrderReques
 		ID:            r.OrderId,
 		Side:          order.Side(r.Side),
 		WalletAddress: r.WalletAddress,
+		Pair : currency.NewPairFromStrings(r.Pair.Base,r.Pair.Quote),
```

Comment:
Go formatting issues picked up by our linter: 
![image](https://user-images.githubusercontent.com/9391715/76469175-d35d9980-6441-11ea-8889-1e67647ac4bb.png)

Please run `gofmt -w rpcserver.go` & push up the fix.